### PR TITLE
Add Sentry integration

### LINKED ISSUE #3: Sentry integration
Configured with an ENV.

### PR DESCRIPTION
**Pull request by papey**
Closes #3

### CODE REVIEW

**Comment by papey** [comment]
LGTM.

File: web/admin.go [hunk 1]
```diff
@@ -53,6 +54,12 @@ func Admin(a *API, w http.ResponseWriter, r *http.Request) {
 
 		w.WriteHeader(http.StatusOK)
 		w.Write(data)
+	case "/sentry/hidden/test":
+		if flag.Lookup("test.v") == nil {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			panic("Testing is panic is handled correctly by Sentry")
```
Comment (athoune) [inline web/admin.go]: what happens if the url is called, and Sentry unconfigured ?

**Review by athoune — CHANGES_REQUESTED** [review]
does test url without Sentry configuration crash the service ?

File: web/admin.go (hunk 1 above)
Comment (papey) [reply web/admin.go]: It just panics but note that [the endpoint is plug only when running `go test`](https://github.com/factorysh/gitlab-log-reader/pull/11/files/dbc7f540b2667656a3c1014ec8dd5d195ecc96d0#diff-f9e6e665e2f336ec441606ba0a67542948fb6a1411893df6783cb8b722589703R58)

**Pull request merged**